dibi/libs/DibiFluent.php

```diff
@@ -315,7 +315,7 @@ public function execute($return = NULL)
 	 */
 	public function fetch()
 	{
-		if ($this->command === 'SELECT') {
+		if ($this->command === 'SELECT' && $this->_export('limit') === NULL) {
```

Comment:
It may be null, but never mind, I have created even better solution here in #188